www/src/pages/contributing/__tests__/stub-list.js

```diff
@@ -38,20 +38,20 @@ import theme from "../../../../src/gatsby-plugin-theme-ui"
 import StubList from "../stub-list"
 
 let location
-beforeEach(() => {
-  useStaticQuery.mockReturnValueOnce({
-    site: {
-      siteMetadata: {
-        title: `GatsbyJS`,
-      },
-    },
-  })
-  location = {
-    pathname: `/contributing/stub-list`,
-  }
-})
 
 describe(`StubList`, () => {
+  beforeAll(() => {
```

Comment:
This was necessary because the update to `theme-ui` seems to be causing some extra re renders. 

Because `useStaticQuery` was initially only mocked once, the re render would receive `undefined` from the hook and break the test. 

@jxnblk Any idea what the update in `theme-ui` changed to cause these extra re-renders?

Reply:
I wasn't sure how Renovate would handle the 0.2 to 0.3 update, but it should probably be treated like a small breaking change -- can `theme-ui` be pinned to 0.2 and handled separately?

I'm not entirely sure why it would cause an extra render, but it might be due to the changes in the `ThemeProvider` component's API

Reply:
@jxnblk Yup, we've decided to pin it till Renovate handles these gracefully (there's an open issue at https://github.com/renovatebot/renovate/issues/4826)